src/app/users/views/selectOrganisationRedesigned.ejs

```diff
@@ -15,55 +15,64 @@
                     <fieldset class="govuk-fieldset">
```

Comment:
if you add id="selectedOrganisation" then the error message in the error summary at the top will link to it

Reply:
You meant this ` model.validationMessages.id="selectedOrganisation"`